base_report_to_printer/report.py

```diff
@@ -42,7 +42,8 @@ def print_document(self, cr, uid, ids, report_name, html=None,
             raise exceptions.Warning(
                 _('No printer configured to print this report.')
             )
-        return printer.print_document(report, document, report.report_type)
+        return printer.with_context(context).print_document(
```

Comment:
I don't think 'must_skip_send_to_printer' option is necessary at this time anymore